.github/workflows/eslint.yml

```
@@ -0,0 +1,27 @@
+name: ESLint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  changed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - id: changed-files
+        uses: tj-actions/changed-files@v19
```

Should use the latest version, if possible

```suggestion
        uses: tj-actions/changed-files@v42
```